src/AddTodoForm.js

```diff
@@ -1,14 +1,27 @@
 import React from "react";
 
 
-function AddTodoForm(){
+function AddTodoForm(props){
```

Comment:
Props aren't destructured, as per the lesson instructions, but it's not a big problem as I see you do destructure the props in `TodoList.js` so you clearly understand what it is and how to do it.

In a professional environment, things like this will usually be decided by the specific company, however make sure to pay attention to things like this if you ever have to do a coding test as part of a job interview. If you miss something out they will assume it's because you don't know how to do it.